.github/ISSUE_TEMPLATE/unmaintained.yaml

```
@@ -0,0 +1,289 @@
+name: Unmaintained component
+description: Create an issue to track the deprecation of a component
+labels: ["help wanted", "unmaintained"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please open an issue using this template if code owners for a component have been unresponsive in 6 weeks. Ensure
```

```suggestion
        Please open an issue using this template if code owners for a component have been unresponsive for at least 6 weeks. Ensure
```
Nit: feel free to drop if this isn't more clear than what's there already.